dtls.c

```diff
@@ -309,8 +309,12 @@ static int janus_dtls_load_keys(const char* server_pem, const char* server_key,
 /* DTLS-SRTP initialization */
 gint janus_dtls_srtp_init(const char* server_pem, const char* server_key) {
 	const char *crypto_lib = NULL;
-#if OPENSSL_VERSION_NUMBER < 0x10100000L || defined(LIBRESSL_VERSION_NUMBER)
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
+#if defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
Won't this need an extra `#endif`? Not sure as I can't try compiling myself right now.